app/src/main/res/layout/scan_result_dialog.xml

```diff
@@ -62,13 +65,13 @@
         android:layout_marginTop="6dp"
         android:paddingHorizontal="10dp"
         android:paddingVertical="6dp"
-        android:background="@drawable/qr_result_background"
```

Comment:
It has already been expressed by @MHShetty that we want to keep a "box" for the result.

My suggestion:
1. Keep this line.
2. Define a new color element (e.g. `qr_result_background`) with the value of:
    - default: @color/material_dynamic_neutral90
    - night: @color/material_dynamic_neutral20
4. Go to `drawables/qr_result_background.xml` and use the newly defined color resource for the color value (e.g. `@color/qr_result_background`).